### DISCUSSION TITLE
Tapparelle con pulsantiera "volante"

### DISCUSSION THREAD

**Discussion by andreabaroni** [indomus/forum#644] [Hardware & Firmware]
Buongiorno a tutti, vi spiego in breve il problema.
- Nuovo appartamento, totalmente da ristrutturare. 
- Abbiamo deciso di aggiungere alla tapparelle dei motori, così da renderle più comode
- Abbiamo "bucato" dal sottotetto per far arrivare la corrente dentro i cassoni delle tapparelle.

Ora il problema è come comandare queste tapparelle, aggiungere la pulsantiera significherebbe dover scendere con i fili dal motore della tapparella fino alla 503, che comporta dover spaccare il muro.

Mi chiedevo quindi se fosse possibile comandare la tapparella con 2 sonoff (uno dentro la pulsantiera e uno sopra attaccato al motore della tapparella) e configurarli in modo che quando uso la pulsantiera il sonoff collegato comunichi con quello attaccato al motore, facendo muovere la tapparella.

Se avete altre idee sono tutto orecchie.

**Comment by indomus** [comment]
Valuta questo: https://indomus.it/componenti/shelly-i3/

**Reply by andreabaroni** [reply]
Quindi mi consiglia di usare shelly i3 collegato alla pulsantiera, mentre alla tapparella collego un shelly 2.5 e poi configuro l'automazione tramite la loro app ?

**Comment by indomus** [comment]
Esattamente, oppure tramite qualsiasi HUB personale che integri i3 tramite MQTT.

**Reply by andreabaroni** [reply]
Gentilissimo! Grazie mille

**Reply by Ciqsky** [reply]
Una cosa ancora migliore? Usi uno shelly 2.5 + un pulsante zigbee a batteria (che dura anche 2 anni) che lo piazzi dove ti è più congeniale. Così non ti serve alcun filo dal pulsante.

**Reply by andreabaroni** [reply]
Si è una possibile soluzione, abbiamo comunque deciso alla fine di portare i cavi della corrente fino alla pulsantiera per sicurezza. Proverò questa implementazione in ogni caso, ho acquistato un telecomando zigbee e un SonoffDualR3 per testarlo.
EDIT: "la volontà di portare i cavi alla pulsantiera non è stata voluta dal sottoscritto ma i muratori che stavano facendo i lavori di ristrutturazione, pensavano servisse inserire una canalina per il motore della tapparella e quindi hanno fatto il lavoro. Meglio perchè mi rende il lavoro più facile, non molto bene per il mio portafoglio :(
